src/parser/jobs/rdm/modules/DisplayOrder.ts

```diff
@@ -0,0 +1,9 @@
+export const DisplayOrder = {
+	DualCast: 1,
+	ComboIssues: 2,
+	InterruptedCasts: 3,
+	WeavingIssues: 4,
+	MeleeCombo: 5,
+	Embolden: 6,
+	Tinctures: 7,
```

Comment:
style nit: For consistency with other jobs, these should use the CONSTANT_VALUE_ALL_CAPS naming pattern

Reply:
updated with suggestion.